src/ixdat/techniques/ec_ms.py

```diff
@@ -303,393 +91,67 @@ def _get_tspan_list(
         print("Following tspans were selected for calibration: " + str(tspan_list))
         return tspan_list
 
-    def siq_ecms_calibration(
-        self, mol, mass, n_el, tspan, faradaic_efficiency=1, tspan_bg=None
-    ):
-        """Calibrate for mol and mass based on one period of steady electrolysis
-
-        Use `spectro_inlets_quantification` package.
-        Args:
-            mol (str): Name of the molecule to calibrate
-            mass (str): Name of the mass at which to calibrate
-            n_el (str): Number of electrons passed per molecule produced (remember the
-                sign! e.g. +4 for O2 by OER and -2 for H2 by HER)
-            faradaic_efficiency (str): Factor to multiply EC flux by to correct if
-                product is not produced at 100% FE. Also works to deconvolute
-                contribution from different isotopes. Defaults to 1.
-            tspan (tspan): The timespan of steady electrolysis
-            tspan_bg (tspan): The time to use as a background
-
-        Return siq.CalPoint: The result of the ecms_calibration
-        """
-        if not plugins.use_siq:
-            raise QuantificationError(
-                "`ECMSMeasurement.siq_ecms_calibration` only works when using "
-                "`spectro_inlets_quantification`"
-                "(`ixdat.options.activate_siq()`). "
-                "For native ixdat MS quantification, use `ecms_calibration`"
-                "instead."
-            )
-        Y = self.integrate_signal(mass, tspan=tspan, tspan_bg=tspan_bg)
-        Q = self.integrate("raw_current", tspan=tspan) * 1e-3
-        n = Q / (n_el * FARADAY_CONSTANT) * faradaic_efficiency
-        F = Y / n
-        cal = plugins.siq.CalPoint(
-            name=f"{mol}@{mass}",
+    @deprecate(
```

Comment:
ok good, you're just deprecating. :D I was worried they would be removed right away

Reply:
Do you need them to remain in their deprecated state beyond 0.3.0? Then we need to change the hard_deprecation date.

Reply:
You did that in the end, didnt you? I think it's good not to change everything and deprecate it at the same time. Hard deprecation at 0.3.1 is better.